brainscore_language/benchmarks/syntaxgym/README.md

```diff
@@ -0,0 +1,72 @@
+SyntaxGym is a unified platform for targeted syntactic evaluation of language models. It supports all steps
+of the evaluation process, from designing test suites to visualizing final results. The goal of SyntaxGym 
+is to make psycholinguistic assessment of language models more standardized, reproducible, and accessible 
+to a wide variety of researchers.  See the [SyntaxGym Website](https://syntaxgym.org/) for more details.
+
+## YAML section for Benchmark Card
+---
+benchmark details:
+  developer: MIT Computational Psycholinguistics Lab
+  date: 2020
+  questions: mitcpllab@gmail.com
+  version: 1.0
+  type: behavioral
+  license: MIT
+  citations: |
+    @inproceedings{hu-etal-2020-systematic,
+    title = "A Systematic Assessment of Syntactic Generalization in Neural Language Models",
+    author = "Hu, Jennifer and Gauthier, Jon and Qian, Peng and Wilcox, Ethan and Levy, Roger",
+    booktitle = "Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics", 
+    year = "2020", url = "https://www.aclweb.org/anthology/2020.acl-main.158", pages = "1725--1744"}
+    @inproceedings{gauthier-etal-2020-syntaxgym, 
+    title = "{S}yntax{G}ym: An Online Platform for Targeted Evaluation of Language Models", 
+    author = "Gauthier, Jon and Hu, Jennifer and Wilcox, Ethan and Qian, Peng and Levy, Roger",
+    booktitle = "Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics: System Demonstrations",
+    year = "2020", url = "https://www.aclweb.org/anthology/2020.acl-demos.10", pages = "70--76"}
+
+experiment:
+  task: Currently either "next_word" or "reading_times"
+  recording: fMRI
+  experiment_card: NA
+  bidirectionality: NA
+  contextualization: NA
+
+data:
```

Comment:
shouldn't this just reference a data plugin with its own card? I guess there can be sub-selects and re-analysis of those data, but many fields are shared

Reply:
it could.  i added a field under data to reference any existing data cards (see benchmark_tutorial.rst)